CONTRIBUTING.md

```diff
@@ -1,7 +1,10 @@
 # Contributing to cargo-mutants
 
-Pull requests are welcome. If the change is not obvious please feel free to open
-a bug or Github discussion first.
+Pull requests are welcome.
+
+If the change is not obvious please feel free to open a bug or Github discussion first.
+
+If you are interested in working on a bug or feature please say so on the bug first to avoid wasted work.
 
 ## Code of Conduct
```

Comment:
The doc just points to the online Rust CoC, so I think I can just leave a pointer here.

I added it under that name because GitHub looks for a file under that name in some places: <https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project>